build/root/BUILD.root

```diff
@@ -80,3 +80,9 @@ genrule(
     cmd = "grep ^STABLE_BUILD_SCM_REVISION bazel-out/stable-status.txt | awk '{print $$2}' >$@",
     stamp = 1,
 )
+
+go_binary(
+    name = "kubernetes",
+    embed = [":go_default_library"],
+    importpath = "k8s.io/kubernetes",
```

Comment:
Humm..Did this cause the bazel build failure?
```
W0205 06:05:35.708] ERROR: /workspace/k8s.io/kubernetes/BUILD.bazel:84:1: in embed attribute of go_binary rule //:kubernetes: rule '//:go_default_library' does not exist. Since this rule was created by the macro 'go_binary_macro', the error might have been caused by the macro implementation in /root/.cache/bazel/_bazel_root/e9f728bbd90b3fba632eb31b20e1dacd/external/io_bazel_rules_go/go/private/rules/wrappers.bzl:69:15
```

Reply:
how did you end up with a `go_binary` rule in this file?